core/webviz/_markdown.py

```diff
@@ -0,0 +1,41 @@
+import jinja2
+import markdown
+from os import path
+from ._page_element import PageElement
+
+
+class Markdown(PageElement):
+    """
+    A page element for adding `markdown`.
+
+    """
+    def __init__(self, md, css_deps=[]):
+        super(Markdown, self).__init__()
+        self._md = md
+        self._css_deps = css_deps[:]
```

Comment:
The css_deps list and `add_css_dep()` function does not need to be here. You only need the `get_css_dep()` as it is implemented now.